starfish/pipeline/features/spots/detector/detect.py

```diff
@@ -29,6 +30,11 @@ def measure_spot_intensity(
         SpotAttributes table containing coordinates and radii of spots
     measurement_function : Callable[[Sequence], Number])
         Function to apply over the spot volumes to identify the intensity (e.g. max, mean, ...)
+    radius_is_gyration : bool
+        if True, indicates that the radius being passed is gyration, which is a function of
```

Comment:
"radius being passed is gyration" -> "radius corresponds to the radius of gyration"